setup.py

```diff
@@ -8,20 +8,28 @@
 from setuptools import setup, find_packages
 
 
-def get_version():
+def load_version():
     """Returns the version found in nilearn/version.py
 
     Importing nilearn is not an option because there may dependencies
     like nibabel which are not installed and setup.py is supposed to
     install them.
     """
-    locals_dict = {}
+    # load all vars into globals, otherwise
+    #   the later function call using global vars doesn't work.
     globals_dict = {}
     with open(os.path.join('nilearn', 'version.py')) as fp:
-        exec(fp.read(), globals_dict, locals_dict)
+        exec(fp.read(), globals_dict)
 
-    return locals_dict['__version__']
+    return globals_dict
 
+
+def is_installing():
+    # Allow command-lines such as "python setup.py build install"
+    return len(sys.argv) > 1 and 'install' in sys.argv
```

Comment:
> setuptools is recommended by the Python Packaging User Guide over distutils.

Yes. This has been a long running argument. It brings certain good
things, and has benefits.

As long as it is not in the standard library, I am against using it. The
reason it has never made in the standard library, although it has strong
proponents, is that it has never met the quality standards corresponding
to the standard library. Unlike pip, which is now in the standard library
for Python 3.5. It is improving, though, and I am hopeful that something
good will come out of it. However, in the mean time, I think that we can
stick to distutils.

> It was used before I changed setup.py for example back in 2012

OK. I am not sure how I missed that :$.